utils/utils.go

```diff
@@ -203,3 +205,10 @@ func GetWorkflowNameAndRunNumber(workflowName string) (string, string) {
 func FormatSessionURI(serverURL string, path string, token string) string {
 	return serverURL + path + "?token=" + token
 }
+
+func LogCmdFlags(cmd *cobra.Command) {
```

Comment:
What do you think about calling this function in the `runE` function of the root command, so that it is automatically done for each command?

Reply:
Nice suggestion, thanks